modules/photometric_calib/src/Reader.cpp

```diff
@@ -0,0 +1,81 @@
+//
+// Created by Nan Yang on 17/6/14.
+//
+
+#include "precomp.hpp"
+#include "opencv2/photometric_calib/Reader.hpp"
+
+namespace cv { namespace photometric_calib{
+
+unsigned long Reader::getNumImages()
+{
+    return (unsigned long)files.size();
+}
+
+void Reader::loadTimestamps(std::string timesFile)
```

Comment:
what is the format of that file ? You should use OpenCV serialization for whatever you do.